docs/Getting-Started.md

```diff
@@ -155,13 +155,8 @@ async function routes (fastify, options) {
     return { hello: 'world' }
   })
 
-  fastify.get('/search/:id', async (request, reply) => {
-    try {
-      return await collection.findOne({ id: request.params.id })
-    } catch (err) {
-      reg.log.error(err)
-      return new Error('Something went wrong')
-    }
+  fastify.get('/search/:id', (request, reply) => {
+    return collection.findOne({ id: request.params.id })
```

Comment:
the point of this example was to show the usage with an async function, hence the `await`. The try-catch was silly, but we need to keep the `await`  and do something with the result of `findOne`. I know that the promise is implicit, but still it'd be helpful to show.